### DISCUSSION TITLE
ueberzug settings for hyprland (maybe useful for the wiki)

### DISCUSSION THREAD

**Discussion by folke** [ibhagwan/fzf-lua#1272] [General]
In order to make ueberuzg(pp) behave on hyprland, it's best to add these settings to your `hyprland.conf`

```
windowrulev2 = float, class:^(ueberzug.*)$
windowrulev2 = noanim, class:^(ueberzug.*)$
windowrulev2 = noborder, class:^(ueberzug.*)$
windowrulev2 = noshadow, class:^(ueberzug.*)$
```

**Comment by ibhagwan** [comment]
Never used hyperland, I would imagine for those that do this is definitely useful info, we can add this to the Wiki if you'd like, it's open edit for all.

**Comment by louwers** [comment]
Looks like https://github.com/seebye/ueberzug is archived.

Probably references should be replaced with https://github.com/jstkdng/ueberzugpp

**Reply by ibhagwan** [reply]
> Looks like https://github.com/seebye/ueberzug is archived.
> 
> Probably references should be replaced with https://github.com/jstkdng/ueberzugpp

Look at the README under dependencies, it’s ueberzugpp

**Reply by louwers** [reply]
Still one reference left in the README :)

**Reply by ibhagwan** [reply]
> Still one reference left in the README :)

You’re welcome to PR it if you wish :)
